modules/exploits/windows/http/softing_sis_rce.rb

```diff
@@ -0,0 +1,316 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'zip'
+require 'metasploit/framework/login_scanner/softing_sis'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::EXE
+  include Msf::Exploit::FileDropper
+  include Msf::Exploit::Remote::HttpClient
+
```

Comment:
```suggestion
  prepend Msf::Exploit::Remote::AutoCheck
```

Reply:
Good stuff! Added in afd4b8af2ea999964e73214d1e83621daf09e962